### ISSUE TITLE
Error: rpc error: code = Unavailable desc = transport is closing

### ISSUE THREAD

**Issue by edmooneyvtx** [terraform-providers/terraform-provider-aws#12106]
Using the following top 10 rules:

```
# https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules

# ------
module "waf-owasp-top-10-rules" {
  source  = "traveloka/waf-owasp-top-10-rules/aws"
  version = "0.2.0"
  # insert the 5 required variables here
  description    = "OWASP WAF"
  environment    = "dev"
  product_domain = "VV"
  service_name   = "VVVVVVVVVVV"
  target_scope   = "regional"
}

resource "aws_wafregional_web_acl" "waf_rules" {
  name        = "waf-vv"
  metric_name = "waf-vv-acl"

  default_action {
    type = "ALLOW"
  }

  rule {
    action {
      type = "BLOCK"
    }

    priority = 1
    rule_id  = "${module.waf-owasp-top-10-rules.rule_group_id}"
  }
}

# waf test abl

resource "aws_lb" "test" {
  name               = "test-lb-tf"
  internal           = false
  load_balancer_type = "application"
  #  subnet_id          = "subnet-0999999999999"

  enable_deletion_protection = false

  # subnets
  subnet_mapping {
    subnet_id = "subnet-09999999999999"
  }

  subnet_mapping {
    subnet_id = "subnet-09999999999999"
  }

  tags = {
    Environment = "test-test"
  }
}


resource "aws_wafregional_web_acl_association" "acl-association" {
  resource_arn = "${aws_lb.test.arn}"
  web_acl_id   = "${aws_wafregional_web_acl.waf_rules.id}"
}

```

**Comment by edmooneyvtx** [comment]
Additionally - here's the state list after running:

**$ terraform state list**
module.waf-owasp-top-10-rules.aws_wafregional_byte_match_set.owasp_02_auth_token_string_set[0]
module.waf-owasp-top-10-rules.aws_wafregional_sql_injection_match_set.owasp_01_sql_injection_set[0]
module.waf-owasp-top-10-rules.aws_wafregional_xss_match_set.owasp_03_xss_set[0]
module.waf-owasp-top-10-rules.random_id.this[0]

[end]

**Comment by hylaride** [comment]
I'm getting similar errors.  Something changed between 2.48.0 (which worked fine) and 2.49.0 (2.50.0 still has the issue).

```
[INFO] Switching to v0.12.21
[INFO] Switching completed
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

module.iam_base.data.template_file.route53_rr_change_policy_template: Refreshing state...
module.jump_server.data.template_file.jump_server_userdata: Refreshing state...
module.s3_remote_state.aws_s3_bucket.backup_state: Refreshing state... [id=REMOVED-tf-backup-state]

Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Unavailable desc = transport is closing
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
Error: rpc error: code = Canceled desc = context canceled
```

**Comment by edmooneyvtx** [comment]
I'm using '* provider.aws: version = "~> 2.50" ' as well.

**Comment by lordz-md** [comment]
Same on 2.59.

**Comment by Kritika2808** [comment]
Working fine with 2.58 as well. 2.59 has broken WAF

**Comment by Kritika2808** [comment]
It will be fixed in 2.60 version https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2590-april-23-2020

**Comment by plbenpeterson** [comment]
Found that is is broken on 2.59 as well

**Comment by maryelizbeth** [comment]
Hi Y'all, 

This issue should be addressed by #13024 which will be released in [2.60.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2600-unreleased) later this week. 

We'll close out this issue; if this persists with 2.60.0 and beyond, please open another GitHub issue to let us know.
